src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/HotResourceSummary.java

```diff
@@ -30,6 +30,13 @@
  * HotResourceSummary contains information such as the name of the hot resource, the current value
  * threshold, etc. It also contains the top K consumers of this particular resource. It is created
  * by some RCAs who work directly on some type of resource(JVM, CPU etc.)
+ *
+ * <p>This object is persisted in SQLite table
+ * Table name : HotResourceSummary
+ *
+ * <p>schema :
+ * | ID(primary key) | Resource Type | Threshold | Value | Avg | Min | Max | Unit Type | Time Period |ID in HotNodeSummary(foreign key)
```

Comment:
If this is the name of the column, can we add the unit of measure to the name of the column so "time_period_seconds" ?

Reply:
done. changed it to time_period_seconds